### PR TITLE
Update Decidim and Ruby version

### PR DESCRIPTION
**Pull request by LauraJaimeC**

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.6.6
+2.7.2
```
Comment (tramuntanal) [inline .ruby-version:1]: Can we upgrade this and the app repositories to latest patch version? I think it's 2.7.5

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## Version 0.2.6
+- Increase minimum Decidim version to v0.24.0
+- Bump Ruby version to 2.7.2
```
Comment (tramuntanal) [inline CHANGELOG.md:5]:
```suggestion
- Bump Ruby version to 2.7.5
```

**Review by tramuntanal — APPROVED** [review]
I approve but remember to update the comment in the CHANGELOG before merging.

File: CHANGELOG.md (hunk 2 above)
Comment (LauraJaimeC) [reply CHANGELOG.md:5]: Ohhh sorry, the page in the browser was not refreshed and I had not seen, I do it in another PR 🙏🏼

Comment (LauraJaimeC) [reply CHANGELOG.md:5]: https://github.com/CodiTramuntana/decidim-verifications-barcelona_energia_census/pull/11

**Pull request merged**